app/controllers/users/invitations_controller.rb

```diff
@@ -21,22 +21,27 @@ def create
       return
     end
 
-    invite_user(organisation)
+    add_user_to_organisation(organisation)
```

Comment:
I didn't mind the invite_user method name.
add_user_to_organisation sounds to me like it's a done deal and no further action is required.

Keen to hear your thoughts.

Reply:
Yeah I don't really mind either way. My thinking here was that we've got  `invitations` for new users and `membership invites` for existing users. These are the two pathways. So rather than rely on a word that could be misconstrued, I went for something super literal which is the creation of the relationship.